app/controllers/admins/sessions_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Admins::SessionsController < Devise::SessionsController
+  layout 'admin'
+  before_action :check_user
+
+  protected
+  def check_user
+    if current_admin
+      flash.clear
+      redirect_to('/admin') and return
+    end
+  end
```

Comment:
https://stackoverflow.com/questions/31943125/how-to-correctly-use-guard-clause-in-ruby